src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/domain/SentenceLengthTest.kt

```diff
@@ -35,7 +36,7 @@ class SentenceLengthTest {
           CourtId("HIGHCT"),
           "offence",
           sentenceLength
-        ).calculateRecallLength()
+        ).calculateRecallLength(RecallType.FIXED)
```

Comment:
Test could use renaming, to include "FIXED" - or extension of parameters to take recallType arg?

Reply:
We only calculate recallLength for FIXED, but will update in next PR